backend/src/controllers/decisionControllers.js

```diff
@@ -30,15 +30,32 @@ const getDecision = (req, res) => {
 };
 
 const postDecision = (req, res) => {
-  const { title, deadline, content, impact, risk, advantage } = req.body;
-
+  const { title, deadline, content, impact, risk, advantage, userId } =
```

Comment:
in the end, the userId won't come from the request body : prefer a variable declaration here until auth is implemented